_layouts/documentation.html

```diff
@@ -26,6 +26,10 @@
           {% if site.temporary_alert %}
             {% include alert.html content=site.temporary_alert %}
           {% endif %}
+
+          {% if page.lead == null %}
+          <h1 class="usa-display padding-bottom-2">{{ page.title }}</h1>
```

Comment:
It looks like `margin` may not work here, which I suspect is what you might have encountered. Padding it is I guess.

(Ideally this would be a situation where we'd not need to customize the spacing at all and rely on `usa-prose` if that's what we're applying on the container)

Reply:
Yeah the container does have `usa-prose` but I really felt like the header was still way too close to the text... generally I kept the headers without any extra spacing but this one seems closer than the others somehow?

Reply:
Yeah I see that too. My comment was more referring to this being an issue with the base design system styles, which we don't need to sort out here. I think it's fine to add padding.

I still think we'd want to increase it from 2 units to 4 units, per the design.